napalm/iosxr/iosxr.py

```diff
@@ -145,7 +145,9 @@ def compare_config(self):
         else:
             return self.device.compare_config().strip()
 
-    def commit_config(self):
+    def commit_config(self, message=None):
```

Comment:
I think `message=""` as it also gives away the expected type.

Reply:
@dbarrosop I don't have any objections to this, if this changed; then the exception would also need to be changed to:

```
if not message:
    raise NotImplementedException
```

Reply:
You would prefer `message=""` instead of `message=None` ?